### ISSUE TITLE
Multiflagellar spermatozoa 

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9832]
**HPO term**
HP:0034309

**Suggested revision and reasons**
The correct labelling for this term is "Macrozoospermia". The parent term should not be "Abnormal sperm tail morphology (HP:0012868)" as it is currently the case, but "Abnormal sperm head morphology (HP:0012865)". The main issue of this condition is that the sperm heads were not divided during spermiogenesis. This only later results in spermatozoa with multiple flagella. 

ORPHA: 137893

**Comment by pnrobinson** [comment]
@MagoWyr the HPO does not encode pathophysiology but tries to stay as close to the actual observations as possible. Assuming there is also Macrozoospermia this should be encoded separately.

**Comment by MagoWyr** [comment]
We can leave these terms separately as they focus on different features of the sperm but they describe exactly the same phenotype, because macrozoospermia is also characterised by multiflagellar spermatozoa and vice versa.

**Comment by pnrobinson** [comment]
@MagoWyr  this is the way the HPO is structured.
One question
Is [Macrocephalic sperm head HP:0025437](https://hpo.jax.org/app/browse/term/HP:0025437) a synonym for Macrozoospermia? If yes, which label should be the primary label? If not, what is the difference?

**Comment by MagoWyr** [comment]
Macrocephalic sperm head and Macrozoospermia are the same. Macrozoospermia is the more common term, so we could use the other one as a synonyme.
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr  thanks, I made Macrozoospermia the primary label and added Macrocephalic sperm head as the synonym
